README.textile

```diff
@@ -25,7 +25,7 @@ After you have your .pem file. Set what host, port, certificate file location on
     APNS.host = 'gateway.push.apple.com' 
     # gateway.sandbox.push.apple.com is default
 
-    APNS.pem  = '/path/to/pem/file'
+    APNS.pems  = { app1: '/path/to/pem/file/of/app1', app2: '/path/to/pem/file/of/app2' }
```

Comment:
this won't be backwards compatible. 
Instead, it might be better to have `pem=` convert it to something with a default app name

`APNS.pem = '/path/to/pem/file'` 
would be the same as somethin glike:
`APNS.pems = { default: '/path/to/pem/file'}`